Orange/widgets/unsupervised/owhierarchicalclustering.py

```diff
@@ -944,8 +944,8 @@ def _update(self):
 
         if distances is not None:
             # Convert to flat upper triangular distances
-            i, j = numpy.triu_indices(distances.X.shape[0], k=1)
-            distances = distances.X[i, j]
+            i, j = numpy.triu_indices(distances.shape[0], k=1)
+            distances = numpy.asarray(distances[i, j])
```

Comment:
Here distances will always be a DistMatrix.  But there is something unnerving about a 'symmetric matrix' with shape (N, ).